webapp/src/components/GenderBadge/GenderBadge.tsx

```diff
@@ -1,14 +1,26 @@
-import { BodyShape } from '@dcl/schemas'
+import React from 'react'
+import { BodyShape, WearableGender } from '@dcl/schemas'
 import classNames from 'classnames'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
-import React from 'react'
 import { isGender, isUnisex as getIsUnisex } from '../../modules/nft/utils'
 import IconBadge from '../AssetPage/IconBadge'
 import { Props } from './GenderBadge.types'
 import styles from './GenderBadge.module.css'
+import { locations } from '../../modules/routing/locations'
 
-const GenderBadge = ({ bodyShapes, withText, onClick }: Props) => {
+const GenderBadge = ({ bodyShapes, withText, assetType, section }: Props) => {
   const isUnisex = getIsUnisex(bodyShapes)
+  const isGenderBodyShape = isGender(bodyShapes, BodyShape.MALE)
+
+  const href = React.useMemo(() => locations.browse({
```

Comment:
We could just import `useMemo` right?

```suggestion
  useMemo(() => {
```